### PR TITLE
Added links for WebApp & Notification Health metrics

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation for:
- https://github.com/mattermost/mattermost/pull/26979
- https://github.com/mattermost/mattermost/pull/26800
- https://github.com/mattermost/mattermost/pull/26942

@hmhealey - This PR is missing 2 details from you: the Grafana dashboard URL, and a summary of what the metrics include.

### CODE REVIEW

File: source/scale/deploy-prometheus-grafana-for-performance-monitoring.rst [hunk 1]
```diff
@@ -100,20 +100,24 @@ Install Grafana
...
 
-See `this Grafana guide <https://grafana.com/docs/grafana/v7.5/dashboards/export-import/>`__ to learn how to import Grafana dashboards either from the UI or from the HTTP API.
+- `Mattermost Performance Monitoring v2 <https://grafana.com/grafana/dashboards/15582>`_, which contains detailed charts for performance monitoring including application, cluster, job server, and system metrics.
+- `Mattermost Notification Health metrics <https://grafana.com/grafana/dashboards/21305-mattermost-notification-health>`_, which can be used to track different types of notifications sent from Mattermost. Accessing Mattermost Notification Health requires the feature flag ``EnableNotificationMetrics`` to be set to ``true``. System admins can :ref:`disable notification metrics data collection <configure/environment-configuration-settings:enable notification metrics>` through the System Console.
+- `Mattermost WebApp Performance Metrics <https://grafana.com/grafana/dashboards/XXXXX>`_, which contains detailed metrics for client-side performance, including x, y, and z. Accessing Mattermost WebApp Performance Metrics requires the feature flag ``EnableClientMetrics`` to be set to ``true``.
```
Comment (cwarnermm) [inline source/scale/deploy-prometheus-grafana-for-performance-monitoring.rst:117]:
```suggestion
- `Mattermost WebApp Performance Metrics <https://grafana.com/grafana/dashboards/XXXXX>`_, which contains detailed metrics for client-side performance, including x, y, and z. Accessing Mattermost WebApp Performance Metrics requires the feature flag ``ClientMetrics`` to be set to ``true``.
```

File: source/scale/deploy-prometheus-grafana-for-performance-monitoring.rst [hunk 2]
```diff
@@ -100,20 +100,24 @@ Install Grafana
...
+.. tip::
+
+   See `this Grafana guide <https://grafana.com/docs/grafana/v7.5/dashboards/export-import/>`_ to learn how to import Grafana dashboards either from the UI or from the HTTP API.
 
-See `this Grafana guide <https://grafana.com/docs/grafana/v7.5/dashboards/export-import/>`__ to learn how to import Grafana dashboards either from the UI or from the HTTP API.
+- `Mattermost Performance Monitoring v2 <https://grafana.com/grafana/dashboards/15582>`_, which contains detailed charts for performance monitoring including application, cluster, job server, and system metrics.
+- `Mattermost Notification Health metrics <https://grafana.com/grafana/dashboards/21305-mattermost-notification-health>`_, which can be used to track different types of notifications sent from Mattermost. Accessing Mattermost Notification Health requires the feature flag ``EnableNotificationMetrics`` to be set to ``true``. System admins can :ref:`disable notification metrics data collection <configure/environment-configuration-settings:enable notification metrics>` through the System Console.
```
Comment (cwarnermm) [inline source/scale/deploy-prometheus-grafana-for-performance-monitoring.rst:116]:
```suggestion
- `Mattermost Notification Health metrics <https://grafana.com/grafana/dashboards/21305-mattermost-notification-health>`_, which can be used to track different types of notifications sent from Mattermost. Accessing Mattermost Notification Health requires the feature flag ``NotificationMetrics`` to be set to ``true``. System admins can :ref:`disable notification metrics data collection <configure/environment-configuration-settings:enable notification metrics>` through the System Console.
```

**Review by devinbinnie — APPROVED** [review]

File: source/configure/performance-monitoring-configuration-settings.rst
Comment (hmhealey) [inline source/configure/performance-monitoring-configuration-settings.rst:1]: .. config:setting:: perf-enableclientmetrics
  :displayname: Enable Client Performance Monitoring
  :systemconsole: Environment > Performance Monitoring
  :configjson: .MetricsSettings.EnableClientMetrics
  :environment: MM_METRICSSETTINGS_ENABLECLIENTMETRICS

  - **true**: **(Default)** Performance metrics are collected for web and desktop app users.
  - **false**: Performance metrics are not collected for web and desktop app users.

Enable client performance monitoring
~~~~~~~~~~~~~~~~~~~~~~~~~~~

+-----------------------------------------------+---------------------------------------------------------------------------------+
| Enable or disable client performance metrics. | - System Config path: **Environment > Performance Monitoring**                  |
| collection.                                   | - ``config.json setting``: ``".MetricsSettings.EnableClientMetrics": true",``   |
|                                               | - ``config.json setting``: ``".MetricsSettings.EnableClientMetrics": true",``   |
| - **true**: **(Default)** Performance metrics | - Environment variable: ``MM_METRICSSETTINGS_ENABLECLIENTMETRICS``              |
|   are collected for web and desktop app       |                                                                                 |
|   users.                                      |                                                                                 |
| - **false**: Performance metrics are not      |                                                                                 |
|   collected for web and desktop app users.    |                                                                                 |
+-----------------------------------------------+---------------------------------------------------------------------------------+
| See the :doc:`performance monitoring </scale/deploy-prometheus-grafana-for-performance-monitoring>` documentation               |
| to learn more.                                                                                                                  |
+-----------------------------------------------+---------------------------------------------------------------------------------+

File: source/scale/deploy-prometheus-grafana-for-performance-monitoring.rst [hunk 3]
```diff
@@ -100,20 +100,24 @@ Install Grafana
...
 
-See `this Grafana guide <https://grafana.com/docs/grafana/v7.5/dashboards/export-import/>`__ to learn how to import Grafana dashboards either from the UI or from the HTTP API.
+- `Mattermost Performance Monitoring v2 <https://grafana.com/grafana/dashboards/15582>`_, which contains detailed charts for performance monitoring including application, cluster, job server, and system metrics.
+- `Mattermost Notification Health metrics <https://grafana.com/grafana/dashboards/21305-mattermost-notification-health>`_, which can be used to track different types of notifications sent from Mattermost. Accessing Mattermost Notification Health requires the feature flag ``NotificationMetrics`` to be set to ``true``. System admins can :ref:`disable notification metrics data collection <configure/environment-configuration-settings:enable notification metrics>` through the System Console.
+- `Mattermost WebApp Performance Metrics <https://grafana.com/grafana/dashboards/XXXXX>`_, which contains detailed metrics for client-side performance, including x, y, and z. Accessing Mattermost WebApp Performance Metrics requires the feature flag ``ClientMetrics`` to be set to ``true``.
```
Comment (hmhealey) [inline source/scale/deploy-prometheus-grafana-for-performance-monitoring.rst:117]: The URL for the dashboard isn't available yet, but it'll be available after https://github.com/mattermost/mattermost-performance-assets/pull/15 is merged.

As for the other details needed:
```suggestion
- `Mattermost Web App Performance Metrics <https://grafana.com/grafana/dashboards/XXXXX>`_, which contains detailed metrics for client-side performance, including web vitals and Mattermost-specifc metrics. Accessing Mattermost Web App Performance Metrics requires the feature flag ``ClientMetrics`` to be set to ``true``.
```

**Pull request merged**